packages/components/src/Autocomplete/Menu.tsx

```diff
@@ -29,6 +30,8 @@ export function Menu({
   const optionMenuClass = classnames(styles.options, {
     [styles.visible]: visible,
   });
+  const menuItemRefs = useRef<HTMLDivElement[]>([]);
```

Comment:
I don't think we need this ref. We can use `queryBySelector` getting the n-th child in the scrolling function to accomplish the same thing.

Reply:
Good idea!! Now using query by classname instead of setting refs 😄